coretex/qiime2/utils.py

```diff
@@ -107,3 +141,7 @@ def getDenoisedSamples(dataset: CustomDataset) -> List[CustomSample]:
 
 def getPhylogeneticTreeSamples(dataset: CustomDataset) -> List[CustomSample]:
     return dataset.getSamples(isPhylogeneticTreeSample)
+
+
+def getMetadataSamples(dataset: CustomDataset) -> CustomSample:
```

Comment:
refactor function to return only one metadata sample and rename it appropriately